### PR TITLE
Db up

### PR DESCRIPTION
**Pull request by Gabriele-ele**

### CODE REVIEW

File: src/Application/Services/ItemService.cs
Comment (ghost) [inline src/Application/Services/ItemService.cs:1]: This changes should be reverted

File: src/DBEditor/ScriptsSQL/CreateTables/001CreateDB.sql
Comment (ghost) [inline src/DBEditor/ScriptsSQL/CreateTables/001CreateDB.sql:1]: Unnecessary script, line above creates DB which is in connection string
`EnsureDatabase.For.PostgresqlDatabase(dbConnectionString);`

File: src/DBEditor/ScriptsSQL/CreateTables/003CreateTables.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    password character varying(50) COLLATE pg_catalog."default" NOT NULL,
+
+    CONSTRAINT pkey_users  PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS public.items
+(
+    key character varying(50) COLLATE pg_catalog."default" NOT NULL UNIQUE,
+    expiration_period int NOT NULL,
+
```
Comment (ghost) [inline src/DBEditor/ScriptsSQL/CreateTables/003CreateTables.sql]: Add additional column "expiration_date"

File: src/DBEditor/ScriptsSQL/CreateTables/003CreateTables.sql
Comment (ghost) [inline src/DBEditor/ScriptsSQL/CreateTables/003CreateTables.sql:1]: Different tables should be in different scripts

File: src/Domain/Entities/ItemEntity.cs
Comment (ghost) [inline src/Domain/Entities/ItemEntity.cs:1]: This changes should be reverted

File: src/WebAPI/Controllers/ItemController.cs
Comment (ghost) [inline src/WebAPI/Controllers/ItemController.cs:1]: This changes should be reverted

File: src/DBEditor/ScriptsSQL/CreateTables/003CreateTables.sql (hunk 1 above)
Comment (arturascervinskas) [inline src/DBEditor/ScriptsSQL/CreateTables/003CreateTables.sql]: also add "value" column

Comment (arturascervinskas) [reply src/DBEditor/ScriptsSQL/CreateTables/003CreateTables.sql]: I think we should rename the "expiration_period" to "expiration_date" and not add an additional column.

File: src/DBEditor/ScriptsSQL/CreateTables/003CreateItemsTables.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE IF NOT EXISTS public.items
+(
+    key character varying(50) COLLATE pg_catalog."default" NOT NULL UNIQUE,
+    expiration_period int NOT NULL,
+    expiration_date date,
```
Comment (arturascervinskas) [inline src/DBEditor/ScriptsSQL/CreateTables/003CreateItemsTables.sql]: change to expiration_date TIMESTAMP

File: src/DBEditor/ScriptsSQL/CreateTables/003CreateItemsTables.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
```
Comment (arturascervinskas) [inline src/DBEditor/ScriptsSQL/CreateTables/003CreateItemsTables.sql:7]: add column value TEXT

File: src/DBEditor/Program.cs [hunk 4]
```diff
@@ -12,7 +12,7 @@ static bool ExecuteScripts(string dbConnectionString, string scriptFilter)
         var upgrader =
             DeployChanges.To
                 .PostgresqlDatabase(dbConnectionString)
-                .WithScriptsEmbeddedInAssembly(Assembly.GetExecutingAssembly(), name => name.Contains(scriptFilter))
```
Comment (arturascervinskas) [inline src/DBEditor/Program.cs]: revert changes

File: src/DBEditor/Program.cs [hunk 5]
```diff
@@ -26,7 +26,7 @@ static bool ExecuteScripts(string dbConnectionString, string scriptFilter)
             Console.ResetColor();
             return false;
         }
-
+        ;
```
Comment (arturascervinskas) [inline src/DBEditor/Program.cs:29]: do we really need this ? if so resolve

**Review by ghost — APPROVED** [review]

**Review by arturascervinskas — APPROVED** [review]

**Pull request merged**